### DISCUSSION TITLE
Podcasts?

### DISCUSSION THREAD

**Discussion by TheWealthFoundation** [Shabinder/SpotiFlyer#195] [Ideas]
Is there a way to download podcast episodes?  I've tried messing with the link structure, but only song/album/artist links work.  Thanks!

**Comment by Shabinder** [comment]
I have no trusted sources to fetch Podcasts/Episodes From.... like we can get songs from other platforms like Youtube Music etc but I still need to find and test such sources for Episodes/Podcasts
